src/frontend/komponenter/Fagsak/Vedtak/VedtakBegrunnelserTabell/VedtaksperioderMedBegrunnelser/VedtaksperiodeMedBegrunnelserPanel.tsx

```diff
@@ -28,57 +24,13 @@ const VedtaksperiodeMedBegrunnelserPanel: React.FC<IProps> = ({
     const { erPanelEkspandert, onPanelClose, genererteBrevbegrunnelser } =
         useVedtaksperiodeMedBegrunnelser();
 
-    const ugyldigeReduksjonsteksterForÅTriggeFritekst: Begrunnelse[] = [
-        Standardbegrunnelse.REDUKSJON_SATSENDRING,
-        Standardbegrunnelse.REDUKSJON_UNDER_6_ÅR,
-        Standardbegrunnelse.REDUKSJON_UNDER_18_ÅR,
-    ];
-
     const vedtaksperiodeInneholderFramtidigOpphørBegrunnelse =
         vedtaksperiodeMedBegrunnelser.begrunnelser.filter(
             vedtaksBegrunnelser =>
                 (vedtaksBegrunnelser.begrunnelse as Standardbegrunnelse) ===
                 Standardbegrunnelse.OPPHØR_FRAMTIDIG_OPPHØR_BARNEHAGEPLASS
         ).length > 0;
 
-    const vedtaksperiodeInneholderEtterbetaling3MånedBegrunnelse = () =>
-        vedtaksperiodeMedBegrunnelser.begrunnelser.filter(
-            vedtaksBegrunnelser =>
-                (vedtaksBegrunnelser.begrunnelse as Standardbegrunnelse) ===
-                Standardbegrunnelse.ETTER_ENDRET_UTBETALING_ETTERBETALING
-        ).length > 0;
-
-    const vedtaksperiodeInneholderBegrunnelseSomStøtterFritekst = () => {
-        const alleBegrunnelser = [
-            ...vedtaksperiodeMedBegrunnelser.begrunnelser,
-            ...vedtaksperiodeMedBegrunnelser.eøsBegrunnelser,
-        ];
-        return alleBegrunnelser.some(vedtaksbegrunnelse => vedtaksbegrunnelse.støtterFritekst);
-    };
-
-    const erReduksjonsbegrunnelseSomSkalViseFritekst = (
-        vedtaksBegrunnelse: IRestVedtaksbegrunnelse
-    ) =>
-        !ugyldigeReduksjonsteksterForÅTriggeFritekst.includes(vedtaksBegrunnelse.begrunnelse) &&
-        vedtaksBegrunnelse.begrunnelseType === BegrunnelseType.REDUKSJON;
-
-    const visFritekster = () => {
-        const begrunnelser = [
-            ...vedtaksperiodeMedBegrunnelser.begrunnelser,
-            ...vedtaksperiodeMedBegrunnelser.eøsBegrunnelser,
-        ];
-        return (
-            vedtaksperiodeMedBegrunnelser.fritekster.length > 0 ||
-            (vedtaksperiodeMedBegrunnelser.type !== Vedtaksperiodetype.UTBETALING &&
-                vedtaksperiodeMedBegrunnelser.type !== Vedtaksperiodetype.ENDRET_UTBETALING) ||
-            vedtaksperiodeInneholderEtterbetaling3MånedBegrunnelse() ||
-            vedtaksperiodeInneholderBegrunnelseSomStøtterFritekst() ||
-            begrunnelser.filter(begrunnelse =>
-                erReduksjonsbegrunnelseSomSkalViseFritekst(begrunnelse)
-            ).length > 0
-        );
-    };
```

Comment:
Noe logikk her har aldri vært relevant for KS. Det som er relevant har blitt flyttet til backend.